docs/hub/index.md

```diff
@@ -1,5 +1,7 @@
 # Hugging Face Hub documentation
 
+The Hugging Face Hub is a platform with over 60K models, 6K datasets, and 6K demo apps, all open source and publicly available, in an online platform where people can easily collaborate and build ML together. The Hub works as a central place where anyone can explore, experiment, collaborate and build technology with Machine Learning. Are you ready to join the path towards open source Machine Learning? 🤗
```

Comment:
```suggestion
The Hugging Face Hub is a platform with over 60K models, 6K datasets, and 6K demo apps (Spaces), all open source and publicly available, in an online platform where people can easily collaborate and build ML together. The Hub works as a central place where anyone can explore, experiment, collaborate and build technology with Machine Learning. Are you ready to join the path towards open source Machine Learning? 🤗
```
or similar?

(also, line 73 becomes redundant no?)

Reply:
Sounds good! I would keep line 73 as that is diving a bit more in depth.